lib/symbol-index.coffee

```diff
@@ -103,7 +103,7 @@ class SymbolIndex
       @rebuild()
     else
       for fqn, symbols of @entries
-        if symbols is null and @keepPath(fqn)
+        if symbols is null
```

Comment:
Removing this check definitely seems right. `@keepPath` is supposed to be used to build the `@entries` cache, which obviates the need top check again on iterating over `@entries`.